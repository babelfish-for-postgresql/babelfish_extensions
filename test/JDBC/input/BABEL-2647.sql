CREATE FUNCTION dbo.mstvf_2647() RETURNS @tv TABLE (a int NULL)
AS
BEGIN
	INSERT @tv VALUES(0);
	RETURN;
END;
go

SELECT * from dbo.mstvf_2647();
go

DROP FUNCTION dbo.mstvf_2647;
go
