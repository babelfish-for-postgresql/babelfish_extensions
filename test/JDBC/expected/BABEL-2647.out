CREATE FUNCTION dbo.mstvf_2647() RETURNS @tv TABLE (a int NULL)
AS
BEGIN
	INSERT @tv VALUES(0);
	RETURN;
END;
go

SELECT * from dbo.mstvf_2647();
go
~~START~~
int
0
~~END~~


DROP FUNCTION dbo.mstvf_2647;
go
